client/src/App.js

```diff
@@ -9,13 +9,15 @@ import Footer from "./components/Footer/Footer";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import Error from "./components/Error/Error";
 import Navbar from "./components/Navbar/Navbar";
+import LoginForm from './components/Admin/Login/LoginForm';
 
 function App() {
   return (
     <div className="app-component">
       <Router>
         <Navbar />
         <Switch>
+          <Route path="/admin-login" exact component={LoginForm}/>
```

Comment:
just /admin is enough

Reply:
Agreed. But exposing this path to the user might pose a threat to us.

Reply:
nope! That's not the case, it will be made secure